launchable/test_runners/pytest.py

```diff
@@ -92,4 +104,74 @@ def _pytest_formatter(test_path):
 split_subset = launchable.CommonSplitSubsetImpls(
     __name__, formatter=_pytest_formatter).split_subset()
 
-record_tests = launchable.CommonRecordTestImpls(__name__).report_files()
+
+@click.option('--json', 'json_report', help="use JSON report files produced by pytest-dev/pytest-reportlog", is_flag=True)
+@click.argument('source_roots', required=True, nargs=-1)
+@launchable.record.tests
+def record_tests(client, json_report, source_roots):
+
+    ext = "json" if json_report else "xml"
+    for root in source_roots:
+        match = False
+        for t in glob.iglob(root, recursive=True):
+            match = True
+            if os.path.isdir(t):
+                client.scan(t, "*.{}".format(ext))
+            else:
+                client.report(t)
+
+        if not match:
+            click.echo("No matches found: {}".format(root), err=True)
+            return
+
+    if json_report:
+        client.parse_func = PytestJSONReportParser(client).parse_func
+
+    client.run()
+
+
+"""
+If you want to use --json option, please install pytest-dev/pytest-reportlog. (https://github.com/pytest-dev/pytest-reportlog)
```

Comment:
[nits]
this comment better be written somewhere in README than within code.

Reply:
Exactlly true,

Reply:
added: https://github.com/launchableinc/cli/pull/376/commits/89552787fb794806944deeb9ebedbc62d9d8a634

Reply:
great!